src/locales/zh_CN/pokemon-form.ts

```diff
@@ -0,0 +1,168 @@
+import { SimpleTranslationEntries } from "#app/interfaces/locales";
+
+export const pokemonForm: SimpleTranslationEntries = {
+  // Battle forms
+  "mega": "Mega {{pokemonName}}",
+  "megaX": "Mega {{pokemonName}} X",
+  "megaY": "Mega {{pokemonName}} Y",
+  "primal": "Primal {{pokemonName}}",
+  "gigantamax": "G-Max {{pokemonName}}",
+  "eternamax": "E-Max {{pokemonName}}",
```

Comment:
```suggestion
  "mega": "Mega {{pokemonName}}",
  "megaX": "Mega {{pokemonName}} X",
  "megaY": "Mega {{pokemonName}} Y",
  "primal": "原始回归{{pokemonName}}",
  "gigantamax": "超级巨{{pokemonName}}",
  "eternamax": "无极巨{{pokemonName}}",
```
cn form prefix added